frt-part-two.md

```diff
@@ -0,0 +1,40 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | The Face of Tomorrow: Exploring the Upsides and Downsides of Facial Recognition Technology (FRT) (2/2) |
+| Headline  | FRT where innovation meets controversy. Are you concerned about privacy? |
+| Cypherpunktimes | cypherpunktimes.com/...link |
+| Publish Date | 00 Feb 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - Addressing laws around FRT, Case studies and advancements in FRT technology. Concluding in what the technology may mean for privacy and what we can expect in the future
```

Comment:
The general idea with a thesis statement is to illustrate the point that you want to make and give the reader a first glimpse into the article. Along with the title, the thesis statement is your change to tell the reader that this article is for them and why they should keep reading. 

In no more than two sentence, it should tell the reader what the article is about and the main argument / talking point.

If I were writing this article, here's an example of the thesis statement I'd use:

"FRT laws focus on giving power to enforcement at the expense of personal freedoms and privacies. Is this the technology that facilitates an Orwellian future or is this the innovation that propels humanity to greater heights?"

Reply:
That's a great statement. I will use what you have put and on future articles I'll follow this line.

Reply:
Yeah, we can always brainstorm thesis ideas. It can be tricky getting the right tone. 

The good thing about having a thesis statement is it should help to keep the article focused on achieving this singular goal.